exchanges/order/orders.go

```diff
@@ -1141,9 +1145,9 @@ func StringToOrderStatus(status string) (Status, error) {
 		return PartiallyFilledCancelled, nil
 	case Open.String():
 		return Open, nil
-	case Closed.String():
+	case Closed.String(), "POSITION_CLOSED":
 		return Closed, nil
-	case Cancelled.String(), "CANCELED", "ORDER_CANCELLED":
+	case Cancelled.String(), "CANCELED", "ORDER_CANCELLED", "IOC", "REDUCE_ONLY":
```

Comment:
A reduce only order isn't cancelled. Put this at the wrapper/exchange level because its a gateio oddity.
An IOC order isn't necessarily cancelled. Move it too